tests/PHPUnit/System/expected/test_ArchiveCronTest_preArchivedSegment_noOptions__Live.getLastVisitsDetails_year.xml

```diff
@@ -517,6 +517,116 @@
 	</row>
 	<row>
 		<idSite>1</idSite>
+		<idVisit>26</idVisit>
+		<visitIp>::ffff:123.45.67.89</visitIp>
```

Comment:
That is the result of `inet_ntop(inet_pton('0:0:0:0:0:ffff:7b2d:4359'))`. Perhaps it is shorthand.